security/keycloak-jwt/src/test/java/io/quarkus/ts/openshift/security/keycloak/AbstractSecurityKeycloakOpenShiftIT.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.ts.openshift.security.keycloak;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.KubernetesList;
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.openshift.api.model.DeploymentConfig;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.app.metadata.AppMetadata;
+import io.quarkus.ts.openshift.common.CustomizeApplicationDeployment;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import io.quarkus.ts.openshift.common.injection.WithName;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public abstract class AbstractSecurityKeycloakOpenShiftIT extends AbstractSecurityKeycloakOpenShiftTest {
+
+    static String keycloakUrl;
+
+    // TODO this is pretty ugly, but I'm tired and can't think of a better way at the moment
```

Comment:
I'm still ashamed of writing this code, but I can't think of a better way :-) We should perhaps at least encapsulate it into a `static` method somewhere in the `common` module. Not needed in this PR IMHO.

Reply:
There is already an issue to refactor this code: https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/127

Reply:
Ah, right, sorry I forgot.